apps/console/src/features/secrets/components/add-secret-wizard.tsx

```diff
@@ -322,7 +333,9 @@ const AddSecretWizard: FC<AddSecretWizardProps> = (props: AddSecretWizardProps):
                         }
                         validate={ (value): string | undefined => {
                             const error = secretValueValidator(value);
+
```

Comment:
Do we need a new line here?

Reply:
A blank line is expected here because of ES Lint warnings